src/components/PeopleFilters.tsx

```diff
@@ -1,25 +1,141 @@
-export const PeopleFilters = () => {
+/* eslint-disable prettier/prettier */
+import { Person } from '../types';
+import classNames from 'classnames';
+import { SearchParams } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+import { useEffect } from 'react';
+
+type Props = {
+  people: Person[];
+  setCurrentPeopleList: (filteredPeople: Person[]) => void;
+  searchParams: URLSearchParams;
+  setSearchWith: (params: SearchParams) => void;
+};
+
+enum Sex {
+  All = '',
+  Male = 'm',
+  Female = 'f',
+}
+
+const createCenturieRange = (start: number, end: number) => {
+  const result = [];
+
+  for (let i = start; i <= end; i++) {
+    result.push(i);
+  }
+
+  return result;
+};
+
+const centurieRange = createCenturieRange(16, 20);
+
+export const PeopleFilters: React.FC<Props> = ({
+  people,
+  setCurrentPeopleList,
+  searchParams,
+  setSearchWith,
+}) => {
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
The `getAll` method for `searchParams` will return an array, so the fallback to an empty array is unnecessary. Consider removing `|| []`.